### PR TITLE
Polygon ERC20: Override USDC/USDT name and displaySymbol

### PR DESCRIPTION
**Pull request by paulo-bc**

### CODE REVIEW

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: custody.json [hunk 1]
```diff
@@ -1119,6 +1119,18 @@
       "minWithdrawal": 0
     }
   },
+  {
+    "symbol": "USDT.MATIC",
+    "displaySymbol": "USDT",
```
Comment (paulo-bc) [inline custody.json:1124]: IMO, we should plan moving `displaySymbol` declaration from `custody.json` to each individual `extensions/blockchains/<network>/assets/<id>/info.json` file.
Quite strange that to override displaySymbol I need to add it to 'custody' even tho the currency has no custodial support.

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

**Comment by paulo-bc** [comment]
@smarvasti-bc QQ, why does `MATIC.MATIC` and `USDC.MATIC` has `hwsSettings` in `custody.json`?
From what the readme says, it should be null if the asset does not have HWS support.

File: custody.json (hunk 1 above)
Comment (smarvasti-bc) [reply custody.json:1116]: true but custodial also has a product if it has no Mercury and no CustodialWallet it is not custodial :)

File: custody.json [hunk 2]
```diff
@@ -1101,10 +1097,6 @@
     "type": "ERC20",
     "nabuSettings": {
       "custodialPrecision": 6
-    },
```
Comment (smarvasti-bc) [inline custody.json:1104]: why did you remove these ... maybe worthwhile putting them back manually if the update removed them. I have no idea why @pablojorge put these here int he first instance.

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: custody.json (hunk 2 above)
Comment (paulo-bc) [reply custody.json:1104]: Updated to `hwsSettings: null`.
@pablojorge could you confirm what this should be?

Comment (smarvasti-bc) [reply custody.json:1104]: no idea:)

**Comment by blockchain-concourse-platform** [comment]
[+] Running checks...


-  ⚠️  [BSV, COIN]: No logo
-  ⚠️  [CEUR, CELO_TOKEN]: displayed as: cEUR
-  ⚠️  [CLOUT, COIN]: displayed as: DESO
-  ⚠️  [CLOUT, COIN]: minWithdrawal 2000 -> $0.000 not in the $0.01-$10 USD range
-  ⚠️  [CUSD, CELO_TOKEN]: displayed as: cUSD
-  ⚠️  [LTC, COIN]: minWithdrawal 5460 -> $0.006 not in the $0.01-$10 USD range
-  ⚠️  [PAX, ERC20]: displayed as: USDP
-  ⚠️  [RENBTC, ERC20]: displayed as: renBTC

File: custody.json (hunk 2 above)
Comment (paulo-bc) [reply custody.json]: Confirmed with @pablojorge that `MATIC.MATIC` and `USDC.MATIC` have HWS support. Added `USDT.MATIC` with `hwsSettings: null`

File: custody.json (hunk 1 above)
Comment (pablojorge) [reply custody.json:1124]: It's not a bad idea. The intention was for all `extensions/blockchains/<network>/assets/<id>/info.json` files to have the same info as the original file, since they're overrides, and add extra fields in other files, since they're not "intrinsic" to the chain/token but an extra piece of info we use on our frontends.

**Comment by pablojorge** [comment]
@paulo-bc LGTM!
